src/fullnode.rs

```diff
@@ -314,6 +317,7 @@ impl Fullnode {
             tpu_sockets: node.sockets.tpu,
             broadcast_socket: node.sockets.broadcast,
             role_notifiers: (to_leader_receiver, to_validator_receiver),
+            blob_sender: Some(blob_sender),
```

Comment:
Maybe this would be better owned by the Tvu struct? That way  the exit() function for tvu can take care of dropping this instead of fullnode having to drop it on exit

Reply:
yes, but doesn't that require tvu::exit() to take a mutable reference to self? Same problem here as well, if we want to drop it in fullnode.

Reply:
yea it does, so it would just move this chunk of logic surround the Option<BlobSender> into the tvu's exit function. The benefit of this is that the current fullnode's exit() will encapsulate the entire shutdown process without having special case logic here in the fullnode that's specific to the TVU. Right now that test is failing because some of the code in the codebase assumes that calling exit() on fullnode exits the tpu/tvu, when in fact with this change, only close() does that.